skopt/searchcv.py

```diff
@@ -587,77 +509,37 @@ def _make_optimizer(self, params_space):
 
         """
 
-        kwargs = self.optimizer_kwargs.copy()
+        kwargs = self.optimizer_kwargs_.copy()
         kwargs['dimensions'] = dimensions_aslist(params_space)
         optimizer = Optimizer(**kwargs)
 
         return optimizer
 
-    def step(self, X, y, space_id, groups=None, n_jobs=1):
+    def _step(self, X, y, space_id, groups=None, n_jobs=1):
         """Generate n_jobs parameters and evaluate them in parallel.
 
         Having a separate function for a single step for search allows to
         save easily checkpoints for the parameter search and restore from
         possible failures.
```

Comment:
should we remove this comment (L521, 522, 523) as well?

Reply:
Ok, I think too that `_step` is mostly here for code clarity now. I don't know how things such as checkpoint would look, so removing.